src/components/Checkbox.tsx

```diff
@@ -29,10 +29,13 @@ const styles = (theme: Theme) =>
     },
   });
 
+  // TODO: @wangannie look into renaming the onChange props
```

Comment:
Since we need 2 onChange functions this is how I ended up naming them, but I'm open to suggestions!

Reply:
Naming is fine here, I like it!